app/web/features/communities/newEvents/EventsPage.tsx

```diff
@@ -45,27 +112,137 @@ const EventsPage = () => {
     }
   };
 
+  const handleShowCancelledClick = () => {
+    setShowCancelled(!showCancelled);
+  };
+
+  const handleFilterIsMyCommunitiesClick = () => {
+    setIsMyCommunities(!isMyCommunities);
+  };
+
+  const handleFilterIsOnlineOnlyClick = () => {
+    setNearMeLocation(undefined);
+    setIsOnlineOnly(!isOnlineOnly);
+  };
+
+  const handleOnChangeAutocomplete = (event: GeocodeResult) => {
+    if (event) {
+      setIsOnlineOnly(false);
+      setNearMeLocation(undefined);
+      setLocationResult(event);
+    } else {
+      setLocationResult("");
+    }
+  };
+
+  const handleFilterNearMeClick = async () => {
+    const user = await getCurrentUser();
+
+    if (nearMeLocation) {
+      setNearMeLocation(undefined);
+    } else {
+      setIsOnlineOnly(false);
+      setLocationResult("");
+      setNearMeLocation(new LngLat(user.lng, user.lat));
```

Comment:
about the near me, as I said, was the original idea in the mockup, but I though about replacing it by the location pill (which will open a pop-up to type location) or the location bar as you suggested (since user can type its location directly having the near me button feels redundant (for me at least)

maybe makes sense to use it and asks the browser the current user's location, we can speak about this in the meeting tomorrow

Reply:
regardless about the button vs searchbar, seems that selecting the results doesn't affect the results grid
![image](https://github.com/user-attachments/assets/fc078621-e2a8-4630-aa01-86fe498a306d)

Reply:
Should be fixed now!